docs/src/main/java/io/quarkus/docs/generation/AssembleDownstreamDocumentation.java

```diff
@@ -466,6 +466,10 @@ private static String rewriteContent(String fileName,
         return content;
     }
 
+    private static String escapeXrefTitleForReplaceAll(String title) {
+        return title.trim().replace("]", "\\\\]");
```

Comment:
And yes, you need that many backslashes :]

Reply:
Great job! :) Backslash magic.